external-crates/move/crates/move-vm-runtime/src/jit/execution/translate.rs

```diff
@@ -915,14 +915,15 @@ fn alloc_function(
     let module_id = module.self_id();
     let is_entry = def.is_entry;
     let (native, def_is_native) = if def.is_native() {
-        (
-            context.natives.resolve(
-                module_id.address(),
-                module_id.name().as_str(),
-                name_ident_str.as_str(),
-            ),
-            true,
-        )
+        if let Ok(native_fun) = context.natives.resolve(
+            module_id.address(),
+            module_id.name().as_str(),
+            name_ident_str.as_str(),
+        ) {
+            (Some(native_fun), true)
+        } else {
+            (None, false)
```

Comment:
```suggestion
            (None, true)
```
Even without the definitions, this should be true -- hence the original code.